admin/create-dumps.sh

```diff
@@ -89,6 +90,14 @@ else
     exit
 fi
 
+# Lock cron, so it cannot be accidentally terminated.
+admin/cron_lock.py lock-cron create-dumps "Creating $DUMP_TYPE dump."
+
+# Trap should not be called before we lock cron to avoid wiping out an existing lock file
+trap on_exit EXIT
+
+sleep 9999
```

Comment:
is this from testing?

Reply:
Sigh, yes. Testing this is... a bit contrived. :(